guides/common/modules/proc_assigning-content-view-environments-to-hosts-by-using-subscription-manager.adoc

```diff
@@ -0,0 +1,38 @@
+[id="assigning-content-view-environments-to-hosts-by-using-subscription-manager"]
+= Assigning content view environments to hosts by using Subscription Manager
+
+To assign content view environments to hosts by using Subscription Manager, perform the following steps:
+
+.CLI procedure
+. Display the available environments and identify which ones to pass in:
++
+[options="nowrap" subs="+quotes"]
+----
+subscription-manager environments --list
```

Comment:
```suggestion
$ subscription-manager environments --list
```
We usually prefix commands either with `$` to denote that users don't need root privileges to run the command, or with `#` to denote that users need root.

Please, check with a SME and add the character to all commands in this module accordingly.

Reply:
All `subscription-manager` commands would need root. (I don't think hammer needs root? but look elsewhere and do whatever we do for all the Hammer commands ;)

Reply:
@bangelic Jeremy says that SM needs root, which means that the prefix should be `#` for all SM commands. Please, fix that.